src/net/JNetReflector/Program.cs

```diff
@@ -0,0 +1,250 @@
+                                                .Replace(AllPackageClasses.ClassStub.CLASS_PLACEHOLDER, mainClass.JVMClassName())
+                                                .Replace(AllPackageClasses.ClassStub.HELP_PLACEHOLDER, mainClass.JavadocUrl(javaDocUrl))
+                                                .Replace(AllPackageClasses.ClassStub.BASECLASS_PLACEHOLDER, mainClass.JVMBaseClassName())
+                                                .Replace(AllPackageClasses.ClassStub.ISABSTRACT_PLACEHOLDER, jClass.IsAbstract ? "true" : "false")
+                                                .Replace(AllPackageClasses.ClassStub.ISCLOSEABLE_PLACEHOLDER, isClassCloseable ? "true" : "false")
+                                                .Replace(AllPackageClasses.ClassStub.ISINTERFACE_PLACEHOLDER, jClass.IsInterface ? "true" : "false")
+                                                .Replace(AllPackageClasses.ClassStub.ISSTATIC_PLACEHOLDER, isClassStatic ? "true" : "false");
+            }
+
+            var singleClassStr = singleClassTemplate.Replace(AllPackageClasses.VERSION_PLACEHOLDER, SpecialNames.VersionPlaceHolder())
+                                                    .Replace(AllPackageClasses.JAR_PLACEHOLDER, jarName)
+                                                    .Replace(AllPackageClasses.NAMESPACE_PLACEHOLDER, mainClass.Namespace())
+                                                    .Replace(AllPackageClasses.ClassStub.CLASS_PLACEHOLDER, mainClass.JVMClassName())
+                                                    .Replace(AllPackageClasses.ClassStub.NESTED_CLASSES_PLACEHOLDER, subClassAutonoumous.ToString());
+
+            var classPath = Path.Combine(rootDesinationFolder, mainClass.Namespace().Replace('.', Path.DirectorySeparatorChar), $"{mainClass.JVMClassName()}.cs");
+
+            if (!dryRun)
+            {
+                if (File.Exists(classPath)) File.Delete(classPath);
+                File.WriteAllText(classPath, singleClassStr);
+            }
+
+            var subClassStr = subClassBlock.ToString();
+            classBlock = classBlock.Replace(AllPackageClasses.ClassStub.NESTED_CLASSES_PLACEHOLDER, subClassStr.Length != 0 ? subClassStr : string.Empty);
+
+            return classBlock;
+        }
+
+        static void ShowHelp(string errorString = null)
+        {
+            var assembly = typeof(Program).Assembly;
+
+
+            Console.WriteLine($"{assembly.GetName().Name} (ver. {assembly.GetName().Version}) - JNet class reflection utility command line interface");
+            Console.WriteLine($"{assembly.GetName().Name} ");
+            Console.WriteLine();
+            if (!string.IsNullOrEmpty(errorString))
+            {
+                Console.WriteLine("Error: {0}", errorString);
+            }
+
+            Console.WriteLine();
+            Console.WriteLine("Examples:");
+            Console.WriteLine(assembly.GetName().Name + " ");
+        }
+    }
```

Comment:
Has to be reviewed. CLIParser has a method to show help, why don't use it?

Reply:
Done